lib/web_ui/test/engine/pointer_binding_test.dart

```diff
@@ -2180,6 +2267,29 @@ mixin _ButtonedEventMixin on _BasicEventContext {
       clientY: clientY,
     );
   }
+
+  html.Event wheel({
+    @required int button,
```

Comment:
nit: AFAIK wheel events should always have `button` as 0. Maybe omit this parameter.

Reply:
Done